app/lib/project/ImportController.js

```diff
@@ -75,8 +76,11 @@ define([
                     return true;
                 }.bind( null, this._master )
             };
-            this._sourceGlyphSet = this._project.getGlyphSet(
```

Comment:
I wonder if we still use `MetapolatorProject.prototype.getGlyphSet` anywhere after this change?